sgmCommon.pri

```diff
@@ -27,7 +27,16 @@ HEADERS += \
     source/beamline/SGM/SGMBypassLadder.h \
     source/beamline/SGM/SGMXASLadder.h \
 	source/ui/SGM/SGMLaddersView.h \
-    source/beamline/SGM/energy/SGMExitSlitSupport.h
+    source/beamline/SGM/energy/SGMExitSlitSupport.h \
+    source/beamline/SGM/SGMVATValve.h \
+    source/beamline/SGM/SGMSampleChamberVacuum.h \
+    source/ui/SGM/SGMVATValveView.h \
+    source/beamline/SGM/SGMTurboPump.h \
+    $$PWD/source/beamline/SGM/SGMSampleChamber.h \
```

Comment:
Not sure! I upgraded my Qt Creator halfway through this project and I think that's when $$PWD started being added to class paths.